app/src/main/res/values/material_colors.xml

```diff
@@ -94,6 +94,8 @@
     <color name="md_blue_500" tools:keep="@color/md_blue_500">#2196F3</color>
     <color name="md_blue_600" tools:keep="@color/md_blue_600">#1E88E5</color>
     <color name="md_blue_700">#1976D2</color>
+    <color name="md_blue_700_50">#801976D2</color>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/43b06fc24f1ddcb819de2e20dacc4fe82bd270b7/app/src/main/res/values/material_colors.xml#L97"></span>The resource <code>R.color.md_blue_700_50</code> appears to be unused</td>
    </tr>
  </tbody>
</table>